apollo-federation/src/query_plan/query_planning_traversal.rs

```diff
@@ -417,14 +418,21 @@ impl<'a: 'b, 'b> QueryPlanningTraversal<'a, 'b> {
                 no_followups = true;
                 break;
             }
+
+            // capture options in statistics
+            let options_count = &self.parameters.statistics.evaluated_plan_options;
+            options_count.set(options_count.get() + followups_for_option.len());
```

Comment:
Nit: you can call `.update(|count| count + followups_for_option.len())` rather than `.set`

Reply:
sadly it is still experimental -> https://doc.rust-lang.org/std/cell/struct.Cell.html#method.update

Reply:
Ah, darn. I missed that :(